Snakefile

```diff
@@ -275,6 +275,13 @@ PARALLEL_LANGUAGE_PAIRS = [
     'en_ru', 'en_sv', 'en_zh-Hans', 'en_zh-Hant'
 ]
 
+# Tatoeba language pairs; constructed manually such that cs_en, en_lv, en_ro, and
+# en_zh-Hant are removed from PARALLEL_LANGUAGE_PAIRS
```

Comment:
This comment is unclear to me -- can you elaborate?

Reply:
PARALLEL_LANGUAGE_PAIRS contain cs_en, en_lv, en_ro, and en_zh-Hant; however,  SOURCE_LANGUAGES['opus/Tatoeba'] don't contain cs, lv, ro, and zh-Hant. So I made a separate list (TATOEBA_LANGUAGE_PAIRS) excluding these languages. 

I now see how my wording is confusing in the comment. Is the following better?
"Construct Tatoeba language pairs manually. TATOEBA_LANGUAGE_PAIRS is a subset of PARALLEL_LANGUAGE_PAIRS; language pairs excluded in it are cs_en, en_lv, en_ro, and en_zh-Hant (as SOURCE_LANGUAGES['opus/Tatoeba'] don't contain cs, lv, ro, and zh-Hant)."